keps/sig-contributor-experience/1553-issue-triage/kep.yaml

```diff
@@ -0,0 +1,20 @@
+title: Issue Triage Workflow and Automation
+authors:
+  - "@justaugustus"
+owning-sig: sig-contributor-experience
+status: provisional
+creation-date: 2020-02-15
+last-updated: 2020-02-15
+reviewers:
+  - "@cblecker"
+  - "@nikhita"
+  - "@thockin"
+approvers:
```

Comment:
contribex TLs should be approvers here, rather than chairs.

Reply:
Will fix up.

Reply:
Fixed.